source/TestCodeHiApp/app/src/main/java/com/fullstackdiv/testcodehiapp/Helper.kt

```diff
@@ -0,0 +1,46 @@
+package com.fullstackdiv.testcodehiapp
+
+import android.graphics.Color
+import java.text.SimpleDateFormat
+import java.util.*
+import java.util.concurrent.ThreadLocalRandom
+
+/**
+ * Created by Angga N P on 10/26/2018.
+ */
+
+class Helper {
```

Comment:
simply use kotlin function extension to make these functions

Reply:
Thanks for your feedback Mr. Oki Wicaksono, this one is global Helper function to manipulate data in every places in this project. That's why i make this a Class. Suggesstion is welcome.